x-pack/plugins/observability_solution/infra/public/components/lens/lens_wrapper.tsx

```diff
@@ -93,7 +93,6 @@ export const LensWrapper = ({
     <div
       css={css`
         position: relative;
-        overflow: hidden;
```

Comment:
Do we know the reason this was originally added? 
If it was intentional, perhaps it was meant to address a specific scenario.

Reply:
This doesn’t give us full certainty, but if you feel confident nothing will break, let’s proceed with the merge!